### ISSUE TITLE
Is authenticating to 2 different APIs possible?

### ISSUE THREAD

**Issue by slawekzachcial** [authts/oidc-client-ts#472]
Hello,

I'm trying to verify a feasibility of a single page application (SPA) that connects to 2 different APIs that are "protected" by the same OIDC identity provider (so 2 different client IDs defined in this IdP). I tried that with [oidc-client-js](https://github.com/IdentityModel/oidc-client-js) and it seems that `UserManager` is scoped to a single client ID, and having 2 `UserManager` instances creates conflicts as they both seem to "compete" for the same session storage.

Does `oidc-client-ts` support having multiple instances of `UserManager` allowing to authenticate to multiple APIs?
If not - is there a different approach that would make it possible, other than creating a single logical API that hides the other APIs?

Thanks!
/Slawek

**Comment by pamapa** [comment]
As long as you you only have **one** client id per `UserManager`, you should be able to have multiple instances of the `UserManager`, all separately configured.

**Comment by slawekzachcial** [comment]
Thanks @pamapa.

Actually, as soon as I wrote my question down I realized it does not make sense.

Client represents the application that interacts, on behalf of the user, with the APIs. Since these APIs are protected by the same Identity Provider, it makes sense that the application has the same Client ID when interacting with each API. It's still good to know that it's possible to have multiple instance of `UserManager`.
